backend_new/src/services/user.service.ts

```diff
@@ -256,12 +260,39 @@ export class UserService {
   /*
     this will update a user or error if no user is found with the Id
   */
-  async update(dto: UserUpdate, jurisdictionName?: string): Promise<User> {
+  async update(
+    dto: UserUpdate,
+    requestingUser: User,
+    jurisdictionName?: string,
```

Comment:
Should this be required? I was suprised to see that an optional param being used as a fallback on line 333

Reply:
updated!